### PR TITLE
feat(vpc/vip): support import in vip_associate

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

support import in vip_associate

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes NONE

**Special notes for your reviewer**:

**Release note**:

```release-note
support import in vip_associate
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccNetworkingV2VIPAssociate_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccNetworkingV2VIPAssociate_basic -timeout 360m -parallel 4
=== RUN   TestAccNetworkingV2VIPAssociate_basic
=== PAUSE TestAccNetworkingV2VIPAssociate_basic
=== CONT  TestAccNetworkingV2VIPAssociate_basic
--- PASS: TestAccNetworkingV2VIPAssociate_basic (216.48s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       216.563s
```

### CODE REVIEW

File: docs/resources/networking_vip_associate.md [hunk 1]
```diff
@@ -45,3 +45,12 @@ In addition to all arguments above, the following attributes are exported:
...
 * `vip_ip_address` - The IP address in the subnet for this vip.
 * `ip_addresses` - The IP addresses of ports to attach the vip to.
+
+## Import
+
+Vip associate can be imported using the `vip_id` and port IDs separated by slashes (no limit on the number of
+port IDs), e.g.
+
+```
+$ terraform import huaweicloud_networking_vip_associate.vip_associated vip_id/port1_id/port2_id
```
Comment (niuzhenguo) [inline docs/resources/networking_vip_associate.md:55]: suggest to use two separaged parameters `vip_id` and `port1/port2/..`

File: huaweicloud/resource_huaweicloud_networking_vip_associate_v2.go [hunk 2]
```diff
@@ -242,3 +247,18 @@ func resourceNetworkingVIPAssociateV2Delete(d *schema.ResourceData, meta interfa
 	d.SetId("")
 	return nil
 }
+
+func resourceNetworkingVIPAssociateV2Import(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	parts := strings.SplitN(d.Id(), "/", -1)
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_networking_vip_associate_v2.go]: suggest to use strings.Split

Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_networking_vip_associate_v2.go]: fixed

**Pull request merged**